tests/internal/runtime/test_runtime_metrics.py

```diff
@@ -73,6 +74,8 @@ def test_one_metric(self):
 
 class TestRuntimeWorker(BaseTracerTestCase):
     def test_tracer_metrics(self):
+        self.previous_runtime = os.getenv("DD_METRICS_RUNTIME")
```

Comment:
This looks like a job for [`patch.dict`](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch.dict)

Reply:
Good call, done